lib/ansible/cli/galaxy.py

```diff
@@ -426,6 +428,7 @@ def execute_init(self):
         Creates the skeleton framework of a role or collection that complies with the Galaxy metadata format.
         Requires a role or collection name. The collection name must be in the format ``<namespace>.<collection>``.
         """
+        raise Exception("abc")
```

Comment:
Was this testing code left in by mistake?

Reply:
Thanks for that, would have been fun if that got in somehow.